ajax/libs/Captionator/0.5/js/captionator-min.js

```diff
@@ -0,0 +1,15 @@
+
+/*
+	Captionator 0.6 [CaptionPlanet]
```

Comment:
What's the reason why it's 0.6 but not 0.5 here? Don't you get it from this url?
https://github.com/cgiffard/Captionator/blob/v0.5/js/captionator-min.js